examples/gpt4-code-interpreter-py/gpt4_code_interpreter/main.py

```diff
@@ -43,9 +43,9 @@
 async def run_code(code: str):
   global session
   # 1. First we need to write the code to a file.
-  await session.filesystem.write("/home/user/index.js", code)
+  await session.filesystem.write("/home/user/main.py", code)
   # 2. Then execute the file with Node.
-  proc = await session.process.start("node /home/user/index.js")
+  proc = await session.process.start("node /home/user/main.py")
```

Comment:
I just realized @yigitkonur , if you want to run Python, you also need to change `node` to `pyhton3` when calling the `process`